dissemin/settings/common.py

```diff
@@ -181,7 +183,6 @@
                     "django.template.context_processors.tz",
                     "django.template.context_processors.request"
                 ),
-                'debug': True
```

Comment:
Template caching is used mainly in papers/publiListItem.html, and it saves quite a lot of queries. I think it would be useful to keep template caching because if we turn it off we might oversee some cache invalidation issues (if we don't invalidate the cache when we change something in the paper, changes will not be seen by the user, so that's bad).